ios/brave-ios/Sources/AIChat/AIChatStrings.swift

```diff
@@ -921,5 +929,23 @@ extension Strings {
       comment:
         "The text displayed on the loading screen when searching for a user query"
     )
+    public static let youMessageTitle = NSLocalizedString(
+      "aichat.youMessageTitle",
+      tableName: "BraveLeo",
+      bundle: .module,
+      value:
+        "You",
+      comment:
+        "The word 'You' as in 'Me', 'Myself', 'I' (the user)."
+    )
+    public static let leoAssistantNameTitle = NSLocalizedString(
+      "aichat.leoAssistantNameTitle",
+      tableName: "BraveLeo",
+      bundle: .module,
+      value:
+        "Leo",
```

Comment:
Is Leo even supposed to be translated?

Reply:
It won't be. It's the same as:
```swift
public static let leoNavigationTitle = NSLocalizedString(
      "aichat.leoNavigationTitle",
      tableName: "BraveLeo",
      bundle: .module,
      value: "Leo AI",
      comment: "The title of the advanced settings view"
    )
```
They don't translate it since it's a name.